resource/tasks.json

```diff
@@ -3051,8 +3051,9 @@
         "maskRange": [1, 255]
     },
     "DailyTask": {
-        "action": "ClickSelf",
-        "roi": [400, 0, 880, 100],
+        "baseTask": "WeeklyTask",
+        "template": "WeeklyTask",
+        "rectMove": [-150, 0, 100, 30],
```

Comment:
Let me get this straight, we basically copy the WeeklyTask check, but click on the "Daily" one, this way there's no need to check for the Daily itself but just Weekly?

Reply:
yep, but we don't know if YJ will change WeeklyTask in the future. If they do, I will choose multiTemplate task instead.